frontend/local/src/containers/patients/index.js

```diff
@@ -147,19 +148,53 @@ class PatientList extends React.Component {
+                                <h3>
+                                    Showing {this.props.numberOfPatients} result{this.props.numberOfPatients > 1 && "s"}
+                                </h3>
+                                <table className="table patients">
+                                    <tbody>
+                                        {_.map(this.props.patientsByInitial, (patients, initial) =>
+                                            patients.map(patient => (
+                                                <ListRow
+                                                    patient={patient}
+                                                    key={patient.patientID}
+                                                    canAddToWaitlist={this.props.canAddToWaitlist}
+                                                    waitlistID={this.state.waitlistID}
+                                                />
+                                            ))
+                                        )}
+                                    </tbody>
+                                </table>
+                            </div>
                         ) : (
-                            <table className="table patients">
-                                <tbody>
-                                    {this.props.patients.map(patient => (
-                                        <ListRow
-                                            patient={patient}
-                                            key={patient.patientID}
-                                            canAddToWaitlist={this.props.canAddToWaitlist}
-                                            waitlistID={this.state.waitlistID}
-                                        />
-                                    ))}
-                                </tbody>
-                            </table>
+                            _.map(this.props.patientsByInitial, (patients, initial) => (
+                                <div key={`initial-${initial}`} className="section">
+                                    <h3>{initial}</h3>
+                                    <table className="table patients">
+                                        <tbody>
+                                            {patients.map(patient => (
+                                                <ListRow
+                                                    patient={patient}
+                                                    key={patient.patientID}
+                                                    canAddToWaitlist={this.props.canAddToWaitlist}
+                                                    waitlistID={this.state.waitlistID}
+                                                />
+                                            ))}
+                                        </tbody>
+                                    </table>
+                                </div>
+                            ))
```

Comment:
this could be made more readable if `: this.state.searchQuery ?` would only change the content's of the `H3` tag`